src/System.Management.Automation/engine/regex.cs

```diff
@@ -50,24 +50,22 @@ public enum WildcardOptions
     /// </summary>
     public sealed class WildcardPattern
     {
-        //
         // char that escapes special chars
-        //
         private const char escapeChar = '`';
 
-        //
         // we convert a wildcard pattern to a predicate
-        //
         private Predicate<string> _isMatch;
 
-        //
+        // chars that are considered special in a wildcard pattern
+        private static readonly char[] s_specialChars = new char[] { '*', '?', '[', ']', '`' };
```

Comment:
```suggestion
        private static readonly char[] s_specialChars = new[] { '*', '?', '[', ']', '`' };
```

Reply:
Yes. I personally don't have preference to `new[]` over `new char[]`. But I guess I will just go with it to mute Codacy :)

Reply:
Something something redundant declarations. Abbreviations are becoming all the rage. :joy: